pystiche_papers/sanakoyeu_et_al_2018/_modules.py

```diff
@@ -0,0 +1,182 @@
+from typing import List, Tuple, Union, cast
+
+import torch
+from torch import nn
+
+from ..utils import ResidualBlock, same_size_padding
+
+__all__ = [
+    "get_padding",
+    "activation",
+    "conv",
+    "ConvBlock",
+    "ConvTransponseBlock",
+    "residual_block",
+]
+
+
+def get_padding(
```

Comment:
Why is this called `get_padding` but the function below `activation` (without the `get`)? We should keep this consistent.